docs/config.md

```diff
@@ -1455,41 +1455,49 @@ Setting this value to zero will disable the limit entirely.
 jj config path --user
 ```
 
-The rest of this section covers the details of where this file can be located.
+On all platforms, the user's global `jj` configurations are loaded in the following
+precedence order (with later configs overriding earlier ones):
 
-On all platforms, the user's global `jj` configuration file is located at either
-`~/.jjconfig.toml` (where `~` represents `$HOME` on Unix-likes, or
-`%USERPROFILE%` on Windows) or in a platform-specific directory. The
-platform-specific location is recommended for better integration with platform
-services. It is an error for both of these files to exist.
+- `$HOME/.jjconfig.toml`
+- `$XDG_CONFIG_HOME/jj/config.toml` (preferred)
+- `$XDG_CONFIG_HOME/jj/conf.d/*.toml`
+
+where `$HOME` represents the user's home directory (`%USERPROFILE%` on Windows),
+and `$XDG_CONFIG_HOME` represents the platform-specific configuration directory.
+The platform-specific location is recommended for better integration with platform
+services.
+
+The files in the `conf.d` directory are loaded in lexicographic order. This allows
+configs to be split across multiple files and combines well
+with [Conditional Variables](#conditional-variables).
 
 | Platform | Value                                              | Example                                                   |
 | :------- | :------------------------------------------------- | :-------------------------------------------------------- |
 | Linux    | `$XDG_CONFIG_HOME/jj/config.toml`                  | `/home/alice/.config/jj/config.toml`                      |
 | macOS    | `$HOME/Library/Application Support/jj/config.toml` | `/Users/Alice/Library/Application Support/jj/config.toml` |
 | Windows  | `{FOLDERID_RoamingAppData}\jj\config.toml`         | `C:\Users\Alice\AppData\Roaming\jj\config.toml`           |
 
-The location of the `jj` config file can also be overridden with the
+The location of the `jj` user config files/directories can also be overridden with the
 `JJ_CONFIG` environment variable. If it is not empty, it will be used instead
-of any configuration file in the default locations. If it is a path to a TOML
+of any configuration files in the default locations. If it is a path to a TOML
 file, then that file will be loaded instead. If it is a path to a directory,
 then all the TOML files in that directory will be loaded in lexicographic order
-and merged. This allows you to split your configuration across multiple files
-and can combine well with [Conditional Variables](#conditional-variables).
+and merged. Multiple paths can be specified by separating them with a
+platform-specific path separator (`:` on Unix-like systems, `;` on Windows).
 
-For example,
-```shell
-env JJ_CONFIG=/dev/null jj log       # Ignores any settings specified in the config file.
+For example, the following could be used to run `jj` without loading any user configs:
+
+```bash
+JJ_CONFIG=/dev/null jj log       # Ignores any settings specified in the config file.
```

Comment:
What do you think about simplifying this to `JJ_CONFIG=`? That seems quite natural now that it's a list.

Reply:
I agree, I'll do that